airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/test/models.py

```diff
@@ -20,6 +20,7 @@ class AirbyteCiPackageConfiguration(BaseModel):
         False,
         description="Flag indicating the mount of the host docker socket to the container running the poe task, useful when the package under test is using dagger",
     )
+    python_versions: List[str] = Field([], description="List of unique python versions to run the poe tasks on")
```

Comment:
```suggestion
    python_versions: List[str] = Field(description="List of unique python versions to run the poe tasks on")
```

This will make the `python_versions` field require and raise a `ValidationError`. I think its better to have validation at this level instead of checking if `python_versions` is set in the `pipeline.py` and raise a `click.UsageError`

Reply:
Oops :). Sorry, I didn't familiarize myself with this part enough. Thanks for the advice.